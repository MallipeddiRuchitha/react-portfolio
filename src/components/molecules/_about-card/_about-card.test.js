import React from 'react';
import { shallow ,mount} from 'enzyme';
import AboutCard from './_about-card';
 
const array=["Spring","React"]
describe("AboutCard component",() =>{
    test("Matches the snapshot",() => {
        const  aboutCard= shallow(<AboutCard heading="Technical Skills" array={array}/>);
        expect(aboutCard).toMatchSnapshot();

    })
    test("about card element is rendered",() => {
        const aboutCard = shallow(<AboutCard heading="Technical Skills" array={array}/>);
        expect(aboutCard.find("#aboutCard").exists()).toEqual(true);
        
    });

  
});