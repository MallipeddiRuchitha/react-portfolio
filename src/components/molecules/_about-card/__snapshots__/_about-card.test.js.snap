// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutCard component Matches the snapshot 1`] = `
<div
  className="makeStyles-list-1"
  id="aboutCard"
>
  <WithStyles(ForwardRef(Typography))
    className="makeStyles-heading-2"
    variant="subtitle1"
  >
    Technical Skills
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Typography))>
    Spring
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Typography))>
    React
  </WithStyles(ForwardRef(Typography))>
</div>
`;
